### PR TITLE
GO-3442: web-clipper doesn't save full page

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
https://linear.app/anytype/issue/GO-3442/web-clipper-doesnt-save-full-page

1. Web clipper doesn't save all images. That's because some web sites use `data-src` attributes instead of `src`
2. So I added support for `data-src` in html converter

### CODE REVIEW

**Review by fat-fellow — APPROVED** [review]

File: core/block/import/markdown/anymark/markdown.go [hunk 1]
```diff
@@ -233,6 +233,15 @@ func getCustomHTMLRules() []html2md.Rule {
 		simpleText, blockquote, italic, code, bdo, div, img, table}
 }
 
+func extractImageSource(src string, ok bool, selec *goquery.Selection) string {
```
Comment (KirillSto) [inline core/block/import/markdown/anymark/markdown.go:236]: It seems we don't need **ok** as input parameter here

**Review by KirillSto — CHANGES_REQUESTED** [review]

File: core/block/import/markdown/anymark/markdown.go (hunk 1 above)
Comment (AnastasiaShemyakinskaya) [reply core/block/import/markdown/anymark/markdown.go]: Fixed

File: core/block/import/markdown/anymark/markdown.go [hunk 2]
```diff
@@ -233,6 +230,19 @@ func getCustomHTMLRules() []html2md.Rule {
 		simpleText, blockquote, italic, code, bdo, div, img, table}
 }
 
+func extractImageSource(selec *goquery.Selection) string {
+	var (
+		src string
+		ok  bool
+	)
```
Comment (KirillSto) [inline core/block/import/markdown/anymark/markdown.go:237]: These declarations are redundunt as we initialize both of these variables in line 238

**Review by KirillSto — APPROVED** [review]

File: core/block/import/markdown/anymark/markdown.go (hunk 2 above)
Comment (AnastasiaShemyakinskaya) [reply core/block/import/markdown/anymark/markdown.go:237]: We need it, because we return src in the end of the function

**Pull request merged**